src/application-navigation/ApplicationNavigationWorkspace.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Workspace, { WorkspaceItem } from '../workspace';
+import ApplicationNavigationWorkspaceItem from './ApplicationNavigationWorkspaceItem';
+
+const propTypes = {
+  /**
+   * Callback function triggering when the workspace size changes.
+   */
+  onSizeChange: PropTypes.func,
+  /**
+   * Callback function triggering when the presentation state changes.
+   */
+  onPresentationStateChange: PropTypes.func,
+  /**
+   * Callback function triggering when the active.
+   */
+  onActiveItemChange: PropTypes.func,
+  /**
+   * The string key linked to the action workspace.
+   */
+  initialActiveItemKey: PropTypes.string.isRequired,
+  /**
+   * Child content to be placed within the main content region.
```

Comment:
I think this comment needs to be updated. Aren't children here, the workspace items? and workspace doesn't have a main content region

Reply:
Updated.